src/material/menu/menu.ts

```diff
@@ -70,6 +70,9 @@ export interface MatMenuDefaultOptions {
 
   /** Whether the menu has a backdrop. */
   hasBackdrop?: boolean;
+
+  /** Close menu when an outside click is detected */
+  closeOnOutsideClick?: boolean;
```

Comment:
I don't think that this should be a configurable option. IMO we should eventually switch to using the outside click handler instead of the backdrop, but it'll probably break some tests.

Reply:
Ah, that makes sense! It might be above my pay grade to fix though, so I'll close this PR.